ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -372,6 +372,10 @@ if [ ! -z "${OPENSHIFT_INSTALL_PROMTAIL_ON_BOOTSTRAP:-}" ]; then
   inject_promtail_service
 fi
 
+echo "install-config.yaml"
+echo "-------------------"
+cat ${SHARED_DIR}/install-config.yaml | grep -v "password\|username\|pullSecret" | tee ${ARTIFACT_DIR}/install-config.yaml
```

Comment:
Why we moved these lines?

Reply:
get the install-config.yaml even create cluster exit with not expected.